web/src/lib/modals/AlbumPickerModal.svelte

```diff
@@ -199,4 +200,22 @@
       >
     {/if}
   </ModalBody>
+  <ModalFooter>
+    <div class="flex justify-around w-full">
+      <div class="flex gap-4">
+        <div class="flex gap-1 place-items-center">
+          <span class="bg-gray-300 dark:bg-gray-500 rounded p-1">
+            <Icon icon={mdiKeyboardReturn} size="1rem" />
+          </span>
+          <Text size="small">to select</Text>
+        </div>
+        <div class="flex gap-1 place-items-center">
+          <span class="bg-gray-300 dark:bg-gray-500 rounded p-1">
+            <Text size="small">CTRL</Text>
```

Comment:
Can you add `class="mono"` and see if that makes it look any better? Also I think this isn't the same size as the other icon, right? Maybe `size="tiny"` would work? Otherwise everything looks good!